ArcGISRuntimeSDKQt_CppSamples/Scenes/GetElevationAtPoint/GetElevationAtPoint.qml

```diff
@@ -0,0 +1,71 @@
+// [WriteFile Name=GetElevationAtPoint, Category=Analysis]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import Esri.Samples 1.0
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+
+Item {
+
+    SceneView {
+        id: view
+        anchors.fill: parent
+    }
+
+    // Declare the C++ instance which creates the scene etc. and supply the view
+    GetElevationAtPointSample {
+        id: model
+        sceneView: view
+    }
+
+    // Background rectangle for elevation display
+    Rectangle {
+        anchors {
+            bottom: parent.bottom
+            left: parent.left
+            right: parent.right
+            margins: 30
+        }
+
+        color: Qt.rgba(0.2, 0.2, 0.2, 0.65);
+        height: childrenRect.height
+
+        // Elevation display text
+        Text {
+            anchors {
+                horizontalCenter: parent.horizontalCenter
+            }
+            color: "white"
+            padding: 15
+            font.pointSize: 32
```

Comment:
I think in some other samples we make use of font metrics to try and get the correct size

Reply:
What do you mean by "correct" size? The text will up/scale down with the containing box, up/down to a certain point size, which must just be arbitrarily decided, unless I'm mistaken?